src/services/AccountService.ts

```
@@ -86,4 +86,12 @@ export class AccountService {
   }
 }
 
+const getLocalStorageItem = (key: string): string | null => {
+  try {
+    return localStorage.getItem(key);
+  } catch (e) {
+    return null;
```

Print the error to the console instead of swallowing it.